WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -0,0 +1,97 @@
+import UIKit
+
+final class WeatherViewController: UIViewController {
+
+    private let cityTextField: UITextField
+    private let temperatureLabel: UILabel
+    private let humidityLabel: UILabel
+    private var landscapeTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
+    private var portraitTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
+
+    init () {
+        cityTextField = UITextField()
+        temperatureLabel = UILabel()
+        humidityLabel = UILabel()
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
I think you can use lazy var. This way you don't need the init.
```
lazy private var cityTextField = UITextField()
```

Reply:
fixed in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/4dc7eeab72cc22c49dae1be2fe5e4e1b82d5d306)